shard.yml

```diff
@@ -90,6 +90,7 @@ dependencies:
   rubber-soul:
     github: placeos/rubber-soul
     version: ~> 1.16
+    branch: service-vers
```

Comment:
Will need to make a new rubber-soul release (don't really want to be dependent on a branch)